.env.sample

```diff
@@ -6,3 +6,4 @@ DB_USER=""
 DB_PASSWORD=""
 DB_NAME=""
 DB_DIALECT="postgres"
+SECRET="eyJhbGciOiJIUzI1NOkV_adQssw5c"
```

Comment:
Generally it's not good idea to keep sensitive information in config files. It's better to consider using some secres manager